articles/introduction-to-deep-learning/index.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/introduction-to-deep-learning/
+title: Introduction to Deep Learning
+description:
+author: judy-nduati
+date: 2020-09-16T00:00:00-13:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/introduction-to-deep-learning/hero.jpg
+    alt:
+---
+Deep Learning (DL) is becoming increasingly popular among IT enthusiasts due to its promising benefits. It is a method that enables image recognition, natural language processing(NLP), and voice recognition to take place. Have you ever wondered how machines function like the human brain? Well, that is thanks to the concept of Deep Learning. Deep Learning is a potent tool, and once you read this article, you will have a better understanding of what DL is and how it works.
+<!--more-->
+### Introduction
+### What is Deep Learning?
+
+![Venn diagram](/engineering-education/introduction-to-deep-learning/ai-ml-dl.jpg)<br>
+
+[*Image source*](https://towardsdatascience.com/cousins-of-artificial-intelligence-dda4edc27b55)
+
+Deep Learning is a subset of machine learning(ML), DL learns features and tasks directly from data such as images, text, or sound.
+
+To take it further, machine learning is a subset of artificial intelligence(AI). AI is the capability of machines and computers to mimic human intelligence and behavior.
+
+AI is accomplished by studying how the human brain operates while trying to solve a problem. ML is a branch of AI that gives machines and computers the ability to learn data without being programmed.
+
+Deep Learning is a machine learning technique that automatically extracts the useful pieces of information or makes decisions using neural networks. Deep Learning is more than a set of tools to solve practical problems. DL was made to imitate how the human brain works while making decisions.
+
+Neural Networks design depends on the composition of the brainpower. Neural Networks are trained to learn models, identify patterns, and arrange non-identical kinds of information as the human brain does.
+
+Deep Learning algorithms strive to reason the same as humans by studying data with a particular [logical framework](https://medium.com/tebs-lab/introduction-to-deep-learning-a46e92cb0022).
+
+### Machine Learning vs. Deep Learning
+Deep Learning and machine learning are both types of AI that classify data and train models.
+
+1. Data presentation in the system. Machine Learning algorithms need structured data to solve data problems. Deep Learning relies on layers of Neural Networks.
+Example: you have several animal images, such as birds and pets. A person would label each image to help an ML algorithm differentiate the images. Deep Learning is a bit complex. This is because using the same dataset. Deep Learning algorithms toil to evaluate variations between categories and classify them.
```

Comment:
Factual error: When you talk about classification, you are talking about supervised learning, which deals with labeled datasets. Therefore, irrespective of machine learning or deep learning, you would end up labeling the images for classification tasks. The difference lies in feature extraction as mentioned in the earlier comment.

Avoid using vague terms such as deep learning algorithms toil to evaluate variations between categories. Emphasizing on what variations you are talking about will enhance the readability.